src/components/Tooltip/Tooltip.tsx

```diff
@@ -36,6 +40,8 @@ export const Tooltip: ChakraComponent<
       className,
       content,
       id,
+      placement = "top",
+      offset = [0, 8],
```

Comment:
This answers my question above, 8 is the default value. I see it's too close to the element when it's `0,0`.